doc/nrf/known_issues.rst

```diff
@@ -1357,90 +1387,110 @@ OCT-2501: Charging over seven hours results in error
+OCT-2569: BIS headset may enter an unwanted state if gateway power is toggled quickly or
+  if headset is moved out of radio range.
+
+  **Workaround:** Reset BIS headset
 
   **Affected platforms:** nRF5340 Audio DK
 
-.. rst-class:: v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
+.. rst-class:: v2-4-0
 
-OCT-2154: USB audio interface does not work correctly on macOS
-  The audio stream is intermittent on the headset side after connecting the gateway to a Mac computer and starting audio stream.
-  This issue occurs sporadically after building the nRF5340 Audio application with the default USB port as the audio source.
+OCT-2585: Initial L-R sync may lock with an offset.
+  The left and right headset may lock as inteded, but there will be a small static
+  time offset between the two headsets.
 
   **Affected platforms:** nRF5340 Audio DK
 
-.. rst-class:: v2-2-0
+.. rst-class:: v2-4-0
+
+OCT-2587: A CIS gateway will try to connect to a BIS gateway if they have the same device name.
 
-OCT-2347: Stream reestablishment issues in CIS
-  In the CIS mode, if a stream is running and the headset is reset, the gateway cannot reestablish the stream properly.
+  **Workaround:** Use different CONFIG_BT_DEVICE_NAME for BIS and CIS.
 
   **Affected platforms:** nRF5340 Audio DK
 
+
 Controller subsystem for nRF5340 Audio
 --------------------------------------
 
 The following known issues apply to the LE Audio subsystem (NET core controller) for nRF5340 used in the nRF5340 Audio application.
 
-.. rst-class:: v2-3-0 v2-2-0
+.. rst-class:: v2-4-0 v2-3-0 v2-2-0
 
 OCX-138: Some conformance tests not passing
    Not all Bluetooth conformance tests cases pass.
 
-.. rst-class:: v2-3-0 v2-2-0
+.. rst-class:: v2-4-0 v2-3-0 v2-2-0
 
 OCX-152: OCX-146: 40 ms ACL interval may cause TWS to be unstable
   There may be combinations of ACL intervals and other controller settings that cause instabilities to connected or true wireless stereo setups.
 
   **Workaround:** Use an alternative ACL connection interval.
 
-.. rst-class:: v2-3-0 v2-2-0
+.. rst-class:: v2-4-0
+
+OCX-153: Cannot create BIG sync after terminate pending BIG sync
+If a pending BIG sync is canceled by sending the LE Broadcast Isochronous Group Sync Estabilished
+ command, then it is impossible for the host to create a new BIG sync afterwards.
+
+.. rst-class:: v2-4-0 v2-3-0 v2-2-0
 
 OCX-155: Larger timestamps than intended
    The timestamps/Service Data Unit references (SDU refs), may occasionally be larger than intended and then duplicated in the next interval.
 
-.. rst-class:: v2-3-0 v2-2-0
+.. rst-class:: v2-4-0 v2-3-0 v2-2-0
 
 OCX-156: PTO is not supported
    The controller does not support Pre-Transmission Offset.
 
-.. rst-class:: v2-3-0 v2-2-0
+.. rst-class:: v2-4-0 v2-3-0 v2-2-0
 
-OCX-157: OCT-140: Interleaved broadcasts streaming issues
+OCX-157: OCX-140: Interleaved broadcasts streaming issues
```

Comment:
Cannot find these OCX Jira issues. We also need another line for the more exact description for all new issues.

Reply:
The controller cannot handle neither broadcasting interleaved BIS ISO packets nor syncing with interleaved BIS ISO packets when the SDU size is more than 100 bytes and retransmission time is 4.